safeeyes/config/lang/cs.json

```diff
@@ -4,57 +4,59 @@
         "language_name_en": "Czech"
     },
     "app_info": {
-        "description": "Safe Eyes protects your eyes from eye strain (asthenopia) by reminding you to take breaks while you're working long hours at the computer."
+        "description": "Safe Eyes chrání vaše oči před následky přetěžování (asthenopie) když dlouho hledíte do obrazovky počítače a to připomínáním potřebných přestávek."
     },
     "exercises": {
         "short_break_close_eyes": "Zavřete oči",
-        "short_break_roll_eyes": "Kroužete očima",
-        "short_break_rotate_clockwise": "Kroužete očima ve směru hodinových ručiček",
-        "short_break_rotate_counter_clockwise": "Kroužete očima proti směru hodinových ručiček",
-        "short_break_blink": "Mrkejte",
-        "short_break_focus_far_distance": "Soustřeďte se na bod v dálce",
-        "short_break_drink_water": "Dejte si nějakou vodu",
+        "short_break_roll_eyes": "Zakroužete očima (několikrát na každou stranu)",
+        "short_break_rotate_clockwise": "Zakroužete očima ve směru hodinových ručiček",
+        "short_break_rotate_counter_clockwise": "Zakroužete očima proti směru hodinových ručiček",
+        "short_break_blink": "Zamrkejte",
+        "short_break_focus_far_distance": "Zaostřete pohled na nějaký vzdálený objekt",
+        "short_break_drink_water": "Napijte se vody",
         "long_break_walk": "Na chvíli se projděte",
-        "long_break_lean_back": "Opřete se do židle a relaxujte"
+        "long_break_lean_back": "Opřete se zády do židle a uvolněte se"
     },
     "messages": {
-        "ready_for_a_break": "Připravte se na přestávku za {} sekund",
+        "audible_alert_disabled": "Nedaří se zapnout zvuková oznamování",
+        "ready_for_a_short_break": "Připravte se na krátkou přestávku za {} sekund",
+        "ready_for_a_long_break": "Připravte se na dlouhou přestávku za {} sekund",
         "disabled_until_restart": "Pozastaveno do restartu",
-        "disabled_until_x": "Pozastaveno na {}",
-        "next_break_at": "Příští přestávka v {}"
+        "disabled_until_x": "Pozastaveno do {}",
+        "next_break_at": "Příští přestávka v {}",
+        "software_required": "Tato funkce vyžaduje instalaci {}"
     },
     "ui_controls": {
         "about": "O aplikaci",
-        "allow_postpone": "Allow postponing the breaks",
-        "audible_alert": "Zvukové upozornění na konec přestávky",
-        "cancel": "Zrušit",
-        "close": "Close",
+        "allow_postpone": "Umožňit odkládání přestávek",
+        "audible_alert": "Zvukové upozornění na konci přestávky",
+        "cancel": "Storno",
+        "close": "Zavřít",
         "disable": "Pozastavit Safe Eyes",
-        "enable": "Zapnout Safe Eyes",
-        "enable_screen_lock": "Po každé dlouhé přestávce uzamknout obrazovku",
+        "disable_keyboard_shortcut": "Ochranná doba po kterou vypnout klávesovou zkratku aby nedošlo k neúmyslenému přeskočení (v sekundách)",
```

Comment:
Může být - jen pochopení toho, proč se vlastně klávesová zkratka vypíná, pak závisí na správném vyhodnocení až druhé poloviny sdělení - otázkou je, zda při "skenování textu" (dnes už málokdo čte slovo od slova) bude dostatečně srozumitelné...

Uznávám, že zkrátit ochranné na ochran. není úplně obvyklé a oko tak na to není vycvičeno. A co tedy znaky ubrat na slově neúmyslného - neúmysl. - i když si čtenář v hlavě nedoplní do potřebného tvaru, už samotné je srozumitelné. Tedy:

Délka (s) ochranného vypnutí kláves. zkratky (prevence neúmysl. přeskočení)

Ale dávám jen jako alternativu - s Vámi navrhovaným řešením nemám problém :)